app/views/orders/confirmation.html.erb

```diff
@@ -0,0 +1,70 @@
+<% subtotal = 0 %>
+
+<h4>Order Placed: <%= @order_items.order.updated_at.strftime("%m/%d/%Y at %R") %></h4>
+<h4>Order Status: <%= @order_items.order.status %></h4>
+
+<table class="table">
+  <tr>
+    <th>Name</th>
+    <th>Price</th>
+    <th>Qty</th>
+    <th>Total</th>
+  </tr>
+  <% @order_items.each do |item| %>
+  <tr>
+    <% product = Product.find(item.product_id) %>
```

Comment:
OrderItem :belongs_to :product, yeah? Skip doing this lookup and just call `@order_item.product` instead.